GDJS/Runtime/variablescontainer.ts

```diff
@@ -206,7 +206,7 @@ namespace gdjs {
 
         networkSyncData.push({
           name: variableName,
-          value: variable.getAsNumberOrString(),
+          value: variable.getValue(),
```

Comment:
Same here, do a check on the type and serialise only children or only the value. Useless calls might seem harmless, but will confuse the reader (how can it be both having a value and children??)